config/initializers/mime_types.rb

```diff
@@ -4,3 +4,4 @@
 # Mime::Type.register "text/richtext", :rtf
 # Mime::Type.register_alias "text/html", :iphone
 Mime::Type.register_alias "text/vnd.graphviz", :gv
+Mime::Type.register "text/markdown", :govspeak
```

Comment:
I would squash this line with the previous commit, as in that commit we are adding the extension `govspeak` to the templates. 
I would also create two tasks to rename Italy and Sweden as they don't use the extension `govspeak`.